en/starter/testing.md

```diff
@@ -16,51 +16,40 @@ This article will walk you through setting up tests in accordance of the XCTest
 
 # Setting up the Tests directory
 1. In the root level of your project, create a folder called `Tests`.
-2. Inside the `Tests` directory you'll need to create a folder for holding your test suite module. It is recommended that it is the same name as your App module.
+2. Inside the `Tests` directory you'll need to create a folder for holding your test suite module. It is recommended that it is the same name as your App module, suffixed with `Tests` (for example: `AppTests`).
 
 # Creating Test Cases
-1. Create a new Swift file in the test suite folder.
+1. Create a new Swift file in the test suite folder (for example: `AppTests/FooTests.swift`)
 2. When setting up this file you'll want to add `@testable import <module-name>` for each of the modules you want to test. (Note the `@testable` keyword allows you to call internal methods in your tests)
```

Comment:
There should be a step between 2. and 3. that says to include `import XCTest` as well.